relay-server/src/actors/store.rs

```diff
@@ -325,20 +465,24 @@ struct UserReportKafkaMessage {
     event_id: EventId,
 }
 
-#[derive(Debug, Serialize)]
+#[derive(Clone, Debug, Serialize)]
 struct SessionKafkaMessage {
     org_id: u64,
     project_id: ProjectId,
     session_id: Uuid,
     distinct_id: Uuid,
+    quantity: u32,
     seq: u64,
     received: f64,
     started: f64,
     duration: Option<f64>,
-    status: SessionStatus,
+    status: &'static str,
     errors: u16,
     release: String,
     environment: Option<String>,
+    user_agent: Option<String>,
+    os: Option<String>,
+    runtime: Option<String>,
```

Comment:
Let's add all new attributes in a follow-up PR and just implement batch payloads based on the current attribute set to reduce impact of this change. Particularly, UA strings are large.